frontend/components/selector/SelectorLanguage.vue

```diff
@@ -54,7 +54,7 @@
                   'group flex w-full [word-spacing:0.5em] items-center rounded-md pl-4 pr-2 py-2 text-sm',
                 ]"
               >
-                {{ locale.name }}
+                {{ getLocaleCode(locale) }}
```

Comment:
This here needs to be `.name`. I can make a similar function for it just so that we're using the same methods of accessing the local info.